src/store/user-store.js

```diff
@@ -1,52 +1,87 @@
 import { create } from "zustand";
 
 const BASE_URL = "http://localhost:3000";
```

Comment:
we can probably put the base url in a config or .env file, just a suggestion tho

Reply:
sounds good. I can make use of the config file you made.